fitnesse/resources/bootstrap-plus/templates/keyboardShortcuts.vm

```diff
@@ -97,6 +97,10 @@
                     <kbd>Ctrl + ,</kbd>
                     <span class="help-key-def">Find current line in context help</span>
                 </li>
+                <li>
+                    <kbd>Ctrl + /</kbd>
+                    <span class="help-key-def">Comment</span>
```

Comment:
Comment / uncomment is denk ik nog net iets beter?

Reply:
Dit is nu aangepast.